src/cascadia/TerminalControl/Resources/en-US/Resources.resw

```diff
@@ -175,4 +175,10 @@
   <data name="TermControl_RendererRetryButton.Content" xml:space="preserve">
     <value>Resume</value>
   </data>
+  <data name="HowToOpenRun.Text" xml:space="preserve">
+    <value>ctrl+click to follow link</value>
+  </data>
+  <data name="HowToOpenRun.FontStyle" xml:space="preserve">
+    <value>Italic</value>
+  </data>

```

Comment:
```suggestion
```

still gotta nuke this so it doesn't conflict :)

Reply:
Oh my bad I misinterpreted what you meant by "it can have properties from both files". Fixed now!